docs/source/tutorials/building_blocks.rst

```diff
@@ -0,0 +1,342 @@
+#######################
+Manim's building blocks
+#######################
+
+This document explains the building blocks of manim and will give you all the
+necessary tools to start producing your own videos.
+
+Essentially, manim puts at your disposal three different concepts that you can
+orchestrate together in order to produce mathematical animations: the
+**mathematical object** (or **mobject** for short) the **animation**, and the
+**scene**.  As we will see in the following sections, each of these three
+concepts is implemented in manim as a separate class: the :class:`.Mobject`,
+:class:`.Animation`, and :class:`.Scene` classes.
+
+.. note:: It is recommended that you read the tutorials :doc:`quickstart` and
+          :doc:`a_deeper_look` before reading this page.
+
+
+********
+Mobjects
+********
+
+Mobjects are the basic building block for all manim animations.  Each class
+that derives from :class:`.Mobject` represents an object that can be displayed
+on screen.  For example, simple shapes such as :class:`.Circle`,
+:class:`.Arrow`, and :class:`.Rectangle` are all mobjects.  More complicated
+constructs such as :class:`.Axes`, :class:`.FunctionGraph`, or
```

Comment:
Yes, this is intentional. Once those modules are added to the reference, these links will work automatically. I think there is no harm in leaving them for now :)

Note: those modules are currently not in the reference because their docstrings are less than great right now ;)

Reply:
Of course, linking them nonetheless is the correct choice.

I would argue, however, that no matter how non-existant the docstrings for these modules are, they should still be included so that the classes can be found in the documentation. The reference manual should be, IMO, as complete as possible.